src/components/Widget/index.js

```diff
@@ -264,6 +273,7 @@ Widget.propTypes = {
 Widget.defaultProps = {
   isChatOpen: false,
   isChatVisible: true,
+  fullScreenMode: false,
```

Comment:
Please remove the dangling comma, this introduced an eslint error, I realise, the project has tons of those errors, but we're aiming to overhaul it.